catalog/src/main/kotlin/com/adevinta/spark/catalog/configurator/samples/buttons/IconButtonsConfigurator.kt

```diff
@@ -85,7 +87,7 @@ private fun IconButtonSample() {
         var intent by remember { mutableStateOf(IconButtonIntent.Main) }
         var contentDescription by remember { mutableStateOf("Content Description") }
 
-        ConfigedIconButton(
+        ConfiggedIconButton(
```

Comment:
nit: `ConfiguredIconButton` would be more appropriate tbh 🙈

Reply:
It's because it's a confit 🥴🙈
_will change it_